packages/training/python/tests/test_archetype_scoring.py

```
@@ -0,0 +1,848 @@
+        """Each archetype should score highest on their specialty."""
+        # Trading-focused trajectory
+        trading_inputs = TrajectoryRewardInputs(
+            final_pnl=500.0, starting_balance=10000.0, end_balance=10500.0,
+            format_score=0.7, reasoning_score=0.6,
+        )
+        trading_metrics = BehaviorMetrics(
+            trades_executed=10, win_rate=0.7, markets_traded=4, total_pnl=500.0,
+        )
+
+        trader_score = archetype_composite_reward(trading_inputs, "trader", trading_metrics)
+        degen_score = archetype_composite_reward(trading_inputs, "degen", trading_metrics)
```

Ruff (0.14.10)

Local variable `degen_score` is assigned to but never used

Remove assignment to unused variable `degen_score`

(F841)